scripts/loadtest/stub_llm_server.py

```
@@ -0,0 +1,107 @@
+def main():
+    parser = argparse.ArgumentParser(description="Minimal OpenAI-compatible stub LLM server")
+    parser.add_argument("--port", type=int, default=9999, help="Port to listen on (default: 9999)")
+    parser.add_argument(
+        "--latency-ms", type=int, default=2000, help="Simulated LLM latency in ms (default: 2000)"
+    )
+    args = parser.parse_args()
+
+    latency_s = args.latency_ms / 1000.0
+    handler = make_handler(latency_s)
+
+    server = ThreadingHTTPServer(("0.0.0.0", args.port), handler)
```

Ruff (0.15.18)

[error] Possible binding to all interfaces

(S104)